docs/en/operation/Journal.md

```diff
@@ -9,29 +9,38 @@ priority: 5
 * Table of Contents
 {:toc}
 
-Alluxio is fault-tolerant: force-killing the system will not lose metadata. To achieve
-this, the master writes edit logs of all metadata changes. On startup,
-a recovering master will read the edit logs to restore itself back to its previous state.
+Alluxio is fault-tolerant: force-killing the system will not lose metadata.
+To achieve this, the master writes edit logs of all metadata changes (e.g., creating files,
+renaming directories).
+On startup, a recovering master will read the edit logs to restore itself back to its previous state.
 We use the term "journal" to refer to the system of edit logs used to support fault-tolerance.
 The purpose of this documentation is to help Alluxio administrators understand and manage the Alluxio journal.
 
 ## UFS Journal vs Embedded Journal
 
-The UFS journal simplifies certain aspects of Alluxio operation, but it relies on 
-an external Zookeeper cluster for coordination, and relies on a UFS for persistent storage. 
-To get reasonable performance, the UFS journal requires a UFS that supports fast streaming writes, 
-such as HDFS or NFS.
-
-The embedded journal does its own coordination and persistent storage, which comes with a limitation on fault tolerance.
-`n` masters using the embedded journal can tolerate only `floor(n/2)` master failures, 
-compared to `n-1` for UFS journal. For example, With `3` masters, UFS journal can tolerate `2` failures, 
-while embedded journal can only tolerate `1`. However, UFS journal depends on Zookeeper, 
+By setting property "`alluxio.master.journal.type`" to "`UFS`" or "`EMBEDDED`" (default since 2.2),
```

Comment:
There are two types of journals that Alluxio is able to use, `EMBEDDED` or `UFS`.
Starting from 2.2, the default journal type is `EMBEDDED`.
This can be changed by setting the property `"alluxio.master.journal.type"` to `"UFS"` instead of `"EMBEDDED"`.